content/concepts/transport.md

```diff
@@ -115,3 +115,98 @@ and some places in the codebase still use the "swarm" terminology.
 {{% /notice %}}
 
 [definition_switch]: /reference/glossary/#switch
+
+## QUIC
+
+QUIC is a new transport protocol that provides an always-encrypted, stream-multiplexed 
+connection built on top of UDP. It started as an experiment by Google on Google Chrome 
+in 2014, and was later standardized by the IETF in 
+[RFC 9000](https://datatracker.ietf.org/doc/html/rfc9000).
+
+### Key challenges with TCP
+
+1. Head-of-line blocking (HoL blocking): TCP is a single byte stream exposed by the kernel, 
+   so streams layered on top of TCP experience HoL blocking.
+
+   {{% notice "info" %}}
+   HoL blocking occurs when a the head packet holds up the other packets in the transmission 
+   queue. 
+   {{% /notice %}}
+
+2. Ossification: Because TCP is unencrypted, middleboxes can inspect and modify
+   TCP header fields and may break unexpectedly when they encounter anything they don’t like.
+
+   {{% notice "info" %}}
+   Middleboxes are intermediary networking devices that can perform special functions to 
+   inspect, transform, and even manipulate network traffic. Examples of middleboxes are 
+   firewalls, NATs, proxies, and load balancers.
+   {{% /notice %}}
+
+3. Handshake inefficiency: the 3-way handshake is inefficient, as it spends 1-RTT on verifying 
+   the client’s address.
+
+   {{% notice "info" %}}
+   TCP requires a 3-way handshake to establish a reliable, bidirectional connection as it intends to.
+   {{% /notice %}}
```

Comment:
(Personal opinion, others might disagree):
For both points (2. & 3.) I don't feel like the _notice_ adds much substance to what's already been said. In particular the notice on point 3 could be removed.

Reply:
I agree here as well, for both notices you mention. Updated